assets/js/index.js

```diff
@@ -0,0 +1,5 @@
+import * as govukFrontend from '/assets/govuk/govuk-frontend.min.js'
```

Comment:
Replacing separate GOVUK/MoJ init files with this: basically what is in template but paths different as we've not moved to `esbuild`